js-basics/js-basics_console-methods/js/index.js

```diff
@@ -3,19 +3,34 @@ console.clear();
 // The exercise starts here!
 
 // All console methods can be found on mdn for reference: https://developer.mozilla.org/en-US/docs/Web/API/console
+//how to create a variable
+//let nameOfVariable = value; 8, "Burak
+//console.log(nameOfVariable)"
 
 // 1. Store the sentence 'Happy coding!' in a variable named 'coding'.
 
+//var coding = 'Happy coding!';
+//console.log(coding);
+const coding = 'Happy coding!';
+
+
 // 2. Display the variable 'coding' as a message in the console.
+console.log(coding);
 
 // 3. Store the sentence 'Happy hacking!' in the variable named 'hacking'.
```

Comment:
console.log like this is more beautiful:    console.log("Happy Codiung: ", coding);